.github/workflows/examples.yml

```
@@ -10,7 +10,7 @@ on:
   workflow_dispatch:
 
 env:
-  CMLIB_VERSION: 1.1.0
+  CMLIB_VERSION: 1.2.1
   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)